sdcm/tester.py

```diff
@@ -1396,36 +1399,37 @@ def stop_resources(self):
 
     def clean_resources(self):
         # pylint: disable=too-many-branches
-        self.log.debug('Cleaning up resources used in the test')
-
-        if self.db_cluster is not None:
-            if self._failure_post_behavior == 'destroy':
-                self.db_cluster.destroy()
-                self.db_cluster = None
-                if self.cs_db_cluster:
-                    self.cs_db_cluster.destroy()
-            elif self._failure_post_behavior == 'stop':
-                for node in self.db_cluster.nodes:
-                    node.instance.stop()
-                self.db_cluster = None
-
-        if self.loaders is not None:
-            if self._failure_post_behavior == 'destroy':
-                self.loaders.destroy()
-                self.loaders = None
-            elif self._failure_post_behavior == 'stop':
-                for node in self.loaders.nodes:
-                    node.instance.stop()
-                self.db_cluster = None
-
-        if self.monitors is not None:
-            if self._failure_post_behavior == 'destroy':
-                self.monitors.destroy()
-                self.monitors = None
-            elif self._failure_post_behavior == 'stop':
-                for node in self.monitors.nodes:
-                    node.instance.stop()
-                self.monitors = None
+        if self.params.get('execute_post_behavior', False):
+            self.log.debug('Cleaning up resources used in the test')
+
+        # if self.db_cluster is not None:
```

Comment:
why is the commented code here?

Reply:
Rewrote this code to support new post_behavior and allow to control clean instances for manual running sct from local hosts